src/content/1.7/basics/installation/configuration.md

```diff
@@ -22,14 +22,11 @@ Forcing the compilation of Smarty will always slow down the loading time of the
 
 ## Displaying error messages
 
-PrestaShop’s default settings prevent the customer to see any server error message or any debugging code.
+PrestaShop’s default settings prevent the customer to see any server error message or any debugging code. You, on the other hand, need this information in order to correct any potential mistake in your code.
```

Comment:
Hi ! I think your contribution is good but we should provide both ways to activate debug mode. Sometimes a user has broken its shop so bad that he cannot access the BO so the "php file" way can be useful too 😄